packages/core/tests/unit/group-info.test.js

```diff
@@ -32,6 +32,10 @@ describe("group-info", () => {
     type Elf @tag(category: "fantasy") {
       name: String!
     }
+
+    type Query {
+      birds: [Bird!]! @doc(category: "animal")
```

Comment:
I would suggest setting the case linked to this PR, i.e. when a query and type have the same name.
```suggestion
      Fish: [Fish!]! @doc(category: "animal")
```